src/app/components/HumidityCard.tsx

```diff
@@ -0,0 +1,24 @@
+import { roundNumber } from "../util/math";
+import WeatherCard from "./WeatherCard";
+import { WiHumidity } from "rocketicons/wi";
+
+interface HumidityCardProps {
+  dew: number;
+  humidity: number;
+}
+
+export default function humidityCard(props: HumidityCardProps) {
```

Comment:
```suggestion
export default function HumidityCard(props: HumidityCardProps) {
```
capitalize function components